recipes/backrefs/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "backrefs" %}
+{% set version = "4.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/backrefs-{{ version }}.tar.gz
+  sha256: a1bbb68f103cf514bcc481b86c40965478d779422fed7cabc5aade318d506df8
+
+build:
+  number: 0
+  skip: true  # [py<=34]
```

Comment:
Any reason this couldn't be `noarch: python`?

Reply:
I'm not %100 sure, the `setup.py` for the package suggests that the generated unicode tables are python-version-specific:

https://github.com/facelessuser/backrefs/blob/master/setup.py#L34-L45

but I haven't done a through audit to determine if this is true or if the generated tables are platform specific.

Is there a mechanism in place to build a `noarch: python`, but test against multiple python versions?

Reply:
Not really. Looking it over, it doesn't look trivial to adapt it to non-version-specific functionality. I guess it can just stay as-is then.